AAUnlimited/MemMods/AAPlay/Events/Loads.cpp

```diff
@@ -75,6 +75,7 @@ DWORD __declspec(noinline) __stdcall CallOrigLoad(DWORD who, void *_this, DWORD
 	CharacterStruct *loadCharacter = (CharacterStruct*)_this;
 	Poser::LoadCharacter(loadCharacter);
 
+	DrawD3D::canRenderDelay(200); // Waiting 200 frames to prevent drawing HUD elements on naked skin
```

Comment:
please elaborate why 200 frames, what problem it solves, and what causes this problem